scss/product/components/_forms.scss

```diff
@@ -72,6 +72,20 @@ select.form-control {
   display: block;
   width: 100%;
 }
+
+//Form inputs
+
+input[type='number']::-webkit-inner-spin-button,
+input[type='number']::-webkit-outer-spin-button {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  margin: 0;
+}
+
+input[type='number'] {
+  -moz-appearance: textfield;
+}
+
```

Comment:
by default we should not remove the default appearance from number input. I can come from a mixin if require.

refer the counter-input component css for example.

Reply:
It was in maya-theme.scss file, I moved it to form.scss. Then we should have an identifier for number inputs wherever it is used and then can use this mixin for that class or ID.

But we don't have any specific identifier for any other input types. Should we have it only for `number` inputs?

Reply:
Fo contact number input instead of using `type=number` we can use `type=tel`. Also, with this, we can avoid changing default appearance from number input.